.npmignore

```diff
@@ -0,0 +1,8 @@
+/screenshots
```

Comment:
it may be easier to whitelist the files to be included in the package using the `file` property in `package.json` rather than blacklisting everything else.
I may throw that in a separate PR.